alchy/model.py

```diff
@@ -126,12 +125,7 @@ class User(Model):
     @declared_attr
     def __table_args__(cls):  # pylint: disable=no-self-argument
         # pylint: disable=no-member
-        args, kargs = merge_declarative_args(cls.__bases__, '__table_args__')
```

Comment:
I can't think of a problem -- though I guess need to be careful not to traverse the ``__mro__`` (or ``mro()``, if you prefer) all the way to ``ModelBase``, or else I think may end up in an infinite loop...